### PR TITLE
Ajout du fichier des régions d'analyse

### PR DESCRIPTION
**Pull request by tahini**
Ajout du geojson avec les régions d'analyse. La région du domicile est calculée dans `home.RA` lorsque la géographie du domicile est modifiée et lorsque les données pré-remplies incluent une géographie de domicile. Les conditionnelles utilisant le `home.RA` devraient maintenant fonctionner.

Mise à jour de la conditionnelle pour `segmentPaidForParking` en ajoutant les conditions sur la matrice motif x RA de la destination.

## Summary by CodeRabbit

* **New Features**
  * Implemented automatic geographic zone identification based on home location to personalize survey content and questions.

* **Bug Fixes**
  * Enhanced conditional logic for location-dependent survey elements with improved geographic data processing and validation.

* **Tests**
  * Added comprehensive test coverage for home location data handling, regional zone determination, and validation of edge cases.

### CODE REVIEW

**Review by kaligrafy — APPROVED** [review]

File: survey/src/survey/common/commonHelpers.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Ce fichier contient des fonctions qui ne dépendent pas d'evolution-frontend
+// ou backend, donc peut être importer autant dans le frontend que dans le
+// backend
+
+import { booleanPointInPolygon } from '@turf/turf';
+import raZones from '../geojson/RA.json';
```
[the remark this answers is not in the document]
Comment (tahini) [reply survey/src/survey/common/commonHelpers.ts:6]: @coderabbitai If this is true, why doesn't it fail? We should either know at startup, test or runtime?

[the remark this answers is not in the document]
Comment (tahini) [reply survey/src/survey/common/commonHelpers.ts:6]: I think the typescript compiler copies the file to the lib/ folder directly, no need to specify it
